settings.gradle

```diff
@@ -103,6 +105,13 @@ dependencyResolutionManagement {
 
             def au = version("au", auVersion)
             library("almostUnified-forge", "com.almostreliable.mods", "almostunified-forge").versionRef(au)
+
+            def xaerosWorldMap = version("xaeros-world-map", xaerosWorldMapVersion)
+            library("xaeros-world-map", "curse.maven", "xaeros-world-map-317780").versionRef(xaerosWorldMap)
+
+            def xaerosMiniMap = version("xaeros-minimap", xaerosMinimapVersion)
+            library("xaeros-minimap", "curse.maven", "xaeros-minimap-263420").versionRef(xaerosMiniMap)
```

Comment:
please use these keys you've set here for the dependencies in `dependencies.gradle` instead of retyping them.